Library/Formula/bacula-fd.rb

```diff
@@ -13,6 +13,29 @@ class BaculaFd < Formula
   depends_on "readline"
   depends_on "openssl"
 
+  def plist; <<-EOS.undent
+    <?xml version="0.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>UserName</key>
+        <string>root</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/bacula-fd</string>
+          <!-- make sure parameter/argument don't contain any spaces! -->
+          <string>-c/usr/local/etc/bacula-fd.conf</string>
```

Comment:
Homebrew won't always be in `/usr/local`. That prefix is our default, but many users choose to install elsewhere.